candi/bashible/common-steps/cluster-bootstrap/070_install_control_plane.sh.tpl

```diff
@@ -12,15 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{{- $experimentalOption := "--patches" -}}
+{{- $experimentalOption := "" -}}
 {{- if semverCompare "<1.22" .kubernetesVersion -}}
-  {{- $experimentalOption = "--experimental-patches" -}}
+  {{- $experimentalOption = "--experimental-patches /var/lib/bashible/kubeadm/patches" -}}
 {{- end }}
 
+mkdir -p /etc/kubernetes/deckhouse/kubeadm/patches/
+cp /var/lib/bashible/kubeadm/patches/* /etc/kubernetes/deckhouse/kubeadm/patches/
 kubeadm init phase certs all --config /var/lib/bashible/kubeadm/config.yaml
 kubeadm init phase kubeconfig all --config /var/lib/bashible/kubeadm/config.yaml
-kubeadm init phase etcd local --config /var/lib/bashible/kubeadm/config.yaml {{ $experimentalOption }} /var/lib/bashible/kubeadm/patches
-kubeadm init phase control-plane all --config /var/lib/bashible/kubeadm/config.yaml {{ $experimentalOption }} /var/lib/bashible/kubeadm/patches
+kubeadm init phase etcd local --config /var/lib/bashible/kubeadm/config.yaml {{ $experimentalOption }}
+kubeadm init phase control-plane all --config /var/lib/bashible/kubeadm/config.yaml {{ $experimentalOption }}
```

Comment:
Why did we remove experimentalOption for k8s>=1.22 ? kubeadm 1.22 and 1.23 has option `--patches`.
I think use experimentalOption is more clear way.